style.css

```diff
@@ -0,0 +1,959 @@
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: #091e42;
+}
+
+.post .server li:hover {
+  color: #fff;
+  background-color: #091e42;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 5px;
+  font-style: normal;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  width: 124px;
+  height: 48px;
+  border: none;
+  background: #f55800;
+  color: #eee;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button:hover {
+  background: #fff5e1;
+  color: #f55800;
+}
+
+.button:active {
+  background: #e05304;
+  color: #f4f1f0;
+}
+
+.frame {
+  display: grid;
+  grid-template-columns: 369px 369px 369px;
+  justify-content: space-between;
+  padding-top: 50px;
+  gap: 24px;
+}
+
+.column1 {
+  background-image: url("./images/Illustration-prject1.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
+  height: 435px;
+  border: 1px solid;
+}
+
+.column1 .button {
+  width: 100%;
+}
+
+.column2 {
+  background-image: url("./images/Illustration-prject2.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-left: 20px;
+}
+
+.frame h2 {
+  width: 337px;
+  height: 81px;
+  padding-top: 180px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.frame p {
+  width: 337px;
+  height: 72px;
+  padding-top: 50px;
+  padding-bottom: 50px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.frame .server {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-bottom: 30px;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  transition: all 2s ease-out 0s;
+}
+
+.frame .server li:hover {
+  color: black;
+  background: #dfe1e6;
+}
+
+.column3 {
+  background-image: url("./images/Illustration-prject3.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-left: 20px;
+}
+
+.column4 {
+  background-image: url("./images/Illustration-prject4.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-left: 20px;
+  height: 435px;
+}
+
+.column5 {
+  background-image: url("./images/Illustration-prject2.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-left: 20px;
+}
+
+.column6 {
+  background-image: url("./images/Illustration-prject3.png");
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-left: 20px;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  padding-top: 130px;
+  padding-left: 100px;
+  background: #1c1a19;
+}
+
```

Comment:
Kindly the styling of your about-section, it is currently overflowing🙂
<img width="764" alt="image" src="https://user-images.githubusercontent.com/73854999/208181403-daca2c1c-60f4-4d0b-8090-3dae2678f1f3.png">